data_steward/tools/generate_combined_dataset.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 set -ex
```

Comment:
nit:  consider adding `set +ex` at the end of the script.  As is, once set the command line will continue to echo until this is turned off.  

opt:  Do you need `set -e`?  How do you debug when the script stops?  It closes my terminal window and I can't debug when `set -e` is set.

Reply:
sure added the line.

Reply:
Thank you!  It makes running this from the command line a lot nicer.